app/models/service_template_terraform_template.rb

```diff
@@ -17,6 +17,9 @@ def self.create_catalog_item(options, _auth_user)
 
     transaction do
       create_from_options(options).tap do |service_template|
+        dialog_ids = service_template.send(:create_dialogs, config_info)
+        config_info.deep_merge!(dialog_ids)
+        service_template.options[:config_info].deep_merge!(dialog_ids)
```

Comment:
Did you mean to do
```suggestion
        config_info.deep_merge!(dialog_ids)
        service_template.options[:config_info] = config_info
```

Reply:
Hey @putmanoj this is I think the last issue outstanding before I can merge

Reply:
I did see your comments,  as this code is 'copy' from ansible, kept this for last for figure if above can done.  Will check today

Reply:
Done